src/components/menus/TertiaryNav/TertiaryNav.tsx

```diff
@@ -31,17 +30,25 @@ export class TertiaryNav extends React.Component<ITertiaryNavProps & IReactCompo
 				</ul>
 				<div className={classnames(styles.TertiaryContent)}>
 					<Switch>
-						{React.Children.map(this.props.children, (child: any) => {
+						{React.Children.map(this.props.children, (child: any, index: number) => {
```

Comment:
LGTM once `index` is removed since it looks like it's unused after the refactor.

Reply:
Got you